lib/matplotlib/table.py

```diff
@@ -726,17 +726,17 @@ def table(ax,
         The cell edges to be drawn with a line. See also
         `~.CustomCell.visible_edges`.
 
+    Returns
+    -------
+    table : `~matplotlib.table.Table`
```

Comment:
remove the name? (... and possibly the description too, which basically adds nothing)